CHANGELOG.next.asciidoc

```diff
@@ -271,6 +271,9 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Set `agent.name` to the hostname by default. {issue}16377[16377] {pull}18000[18000]
 - Add keystore support for autodiscover static configurations. {pull]16306[16306]
 - Add config example of how to skip the `add_host_metadata` processor when forwarding logs. {issue}13920[13920] {pull}18153[18153]
+- Add keystore support for autodiscover static configurations. {pull]16306[16306]
```

Comment:
@ycombinator the changelog didn't make it

Reply:
Thanks. Fixed in 962e32767.